src/frontend.jl

```diff
@@ -48,6 +48,13 @@ other environments.
 plotsize() = Atom.plotsize()
 
 """
+    showprofile()
+
+Show currently collected profile information in the editor.
+"""
+showprofile() = isactive() && Atom.Profiler.profiler()
```

Comment:
`profiler()`? I think the main thing is that it's easy to find through autocomplete.